docs/getting-started.md

```diff
@@ -144,3 +144,32 @@ gbs build
 ```
 
 `gbs build` will execute unit testing as well unlike meson build.
+
+
+## Troubleshooting
+
+### Error 1:
+
+Applications/TransferLearning/Draw_Classification/jni/meson.build:5:0: ERROR: Dependency "glib-2.0" not found, tried pkgconfig and cmake
+
+### Solution: Please install libglib2.0-dev using the following:
+
+sudo apt-get install libglib2.0-dev
```

Comment:
how about enclosing this in bash code block like below:
```bash
sudo apt-get install libglib2.0-dev
```
for this and 2 instructions below as well?

Reply:
Sure, in the next commit, we can update with the code block.